ibm_catalog.json

```
@@ -256,16 +258,52 @@
               "required": false
             },
             {
-              "display_name": "event_notifications_email_list",
-              "key": "en_email_list",
+              "key": "key_protect_plan",
+              "type": "string",
+              "default_value": "tiered-pricing",
+              "description": "The service plan of the Key Protect instance that will be provisioned by this solution. Only used if not supplying `existing_kms_instance_crn`. Learn more [here](https://cloud.ibm.com/docs/key-protect?topic=key-protect-pricing-plan).",
```

For consistency with other DAs
```suggestion
              "description": "The service plan of the Key Protect instance that will be provisioned by this solution. Only used if not supplying `existing_kms_instance_crn`. [Learn more](https://cloud.ibm.com/docs/key-protect?topic=key-protect-pricing-plan).",
```